README.md

```diff
@@ -18,27 +18,55 @@ In the tech section, we provide information about the technology stack, dependen
 
 ### Tech Stack
 
-For the development of Rainbow warriors we made use of the following stack of technologies:
+To develop the Vegelicious app we made use of the following stack of technologies:
 
 - Front-End
- - HTML3
- - CSS5
- - bootstrap5
- - react
- - typescript
+ - [HTML3](https://www.w3schools.com/html/)
+ - [CSS5](https://www.w3schools.com/css/default.asp)
+ - [react](https://www.w3schools.com/react/default.asp)
+ - [typescript](https://www.w3schools.com/typescript/typescript_intro.php)
 
 - Back-End
- - nodejs
+ - [nodejs](https://www.w3schools.com/nodejs/nodejs_intro.asp)
 
 ### The Database Design
-The database was designed to allow CRUD functionality to be available to registered users, when signed in. 
+- The database was designed to allow CRUD functionality to be available to registered users, when signed in.
+- We used mongodb for our nosql database.  Items in the database were:
+"const {
+    name,
+    price,
+    img,
+    unit,
+    expired_date,
+    description,
+    stock,
+    farmer,
+  }"
+
+- The database was set up with full CRUD functionality.
+Path   |   Method	|  Data Requested (json)   |  Function
+---| ---| ---| ---|  
+/signup   |	post |   {email:string, password:string}	|  register an account
+/login   |	post  |		{email:string, password:string}	| log in
+/logout	 | get	|  |		log out
+/profile  |	put	 |	any data changed |	edit profile
+/delete  |	delete	|	None |	delete account
+
+/product |	get	| |	depends on how to display products in the main page	
+/product |	put	|   |	all the data required for editing a product	
+/product | 	delete	|   |	data id 	
+/product/name   |	get	  |  	{name:string}  |	search products by nameS
+
+
 
  
 
 ### Dependencies including deployment
+- The app was deployed to [vercel](https://vercel.com/) 
 
 
 ### Testing
+- testing was done at the start for the project using [jest](https://jestjs.io/)
```

Comment:
I would delete "at the start of the project" since we did not really do that lol